A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/BRK/A320_Neo_BRK.js

```diff
@@ -67,9 +67,14 @@ var A320_Neo_BRK;
             this.rightGauge.addMarker(2.5, true);
             this.rightGauge.addMarker(3, false);
             this.electricity = this.querySelector("#Electricity");
+            this.updateThrottler = new UpdateThrottler(100);
         }
         onUpdate(_deltaTime) {
             super.onUpdate(_deltaTime);
+            _deltaTime = this.updateThrottler.canUpdate(_deltaTime);
+            if (_deltaTime == -1) {
```

Comment:
```suggestion
            if (this.updateThrottler.canUpdate(_deltaTime) === -1) {
```

Reply:
Hmm I think this may be a bit dangerous to do in case somebody later adds some code below that would use _deltaTime - in that case, _deltaTime would be wrong. Wdyt?